app/code/Magento/Store/Console/Command/StoreCreateCommand.php

```diff
@@ -0,0 +1,128 @@
+<?php
+namespace Magento\Store\Console\Command;
+
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ *
+ */
+class StoreCreateCommand extends Command
+{
+    const INPUT_ARGUMENT_NAME = 'name';
+    const INPUT_ARGUMENT_CODE = 'code';
+    const INPUT_ARGUMENT_IS_ACTIVE = 'is_active';
+    const INPUT_ARGUMENT_SORT_ORDER = 'sort_order';
+
+    const INPUT_OPTION_GROUP = 'group_id';
+
+    /**
+     *
+     * @var \Magento\Store\Model\StoreCreateManagement
+     */
+    private $storeCreateManagement;
+
+    /**
+     * @var \Magento\Framework\Filter\FilterManager
+     */
+    private $filterManager;
+
+    /**
+     *
+     * @param \Magento\Store\Model\StoreCreateManagement $storetoreCreateManagement
+     * @param \Magento\Framework\Filter\FilterManager $filterManager
+     * @param string                                  $name
```

Comment:
It's better to avoid that kind of formatting for consistency purpose
```suggestion
     * @param string $name
```